### PR TITLE
add blacklist table to char delete trigger

### PR DESCRIPTION
**Pull request by KnowOne134**

### CODE REVIEW

File: sql/triggers.sql [hunk 1]
```diff
@@ -45,6 +45,7 @@ CREATE TRIGGER char_delete
 	BEFORE DELETE ON chars
 	FOR EACH ROW
 BEGIN
+    DELETE FROM `char_blacklist` WHERE `charid_owner` = OLD.charid;
```
Comment (zynjec) [inline sql/triggers.sql]: `charid_owner`?

And just for consistency, either use tabs or change the whole file to spaces.

Comment (KnowOne134) [reply sql/triggers.sql]: Ah yeah my edit does tabs to spaces. And yes in the blacklist table the charid is charid_owner

Comment (zynjec) [reply sql/triggers.sql]: Right, forgot it was named something different in blacklist.

cOnSIStEncY

Comment (KnowOne134) [reply sql/triggers.sql]: If you want we can do a overhaul on it and change it all to charid?

Comment (zynjec) [reply sql/triggers.sql]: It's fine, just do tabs to spaces and we're golden.

**Pull request merged**